src/odemis/driver/smaract.py

```diff
@@ -1590,9 +1601,42 @@ def _doMoveAbs(self, future, pos):
         max_dur = dur * 2 + 1
         logging.debug("Expecting a move of %f s, will wait up to %g s", dur, max_dur)
 
+<<<<<<< HEAD
```

Comment:
@amuskens I think this does not belong here and is a result of a not completed pull/rebase/merge?

Reply:
Uh oh hahhaa.... let me fix that...